helm/alfresco-content-services/templates/config-dev-log4j-properties.yaml

```diff
@@ -9,6 +9,11 @@ metadata:
 data:
   dev-log4j.properties: |-
   {{- range $key, $val := .Values.repository.extraLogStatements }}
-   log4j.logger.{{ $key }}={{ tpl $val $ }}
+   log4j.logger.{{ $key }}={{ $val }}
+  {{- end }}
+  dev-log4j2.properties: |-
```

Comment:
I would rename the cm and its entries from `dev-` to `custom-` as it's not really development oriented feature.